operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -205,6 +205,8 @@ const (
 	CollectionCOREBPF CollectionMethod = "CORE_BPF"
 	// CollectionNone means: NO_COLLECTION.
 	CollectionNone CollectionMethod = "NoCollection"
+	// CollectionKernelModule means: use KERNEL_MODULE collection.
+	CollectionKernelModule CollectionMethod = "KernelModule"
```

Comment:
Does the source code automatically switch to `eBPF` and print a user warning if `KernelModule` is selected?

If yes, please merge

Reply:
Yes. It does switch to `eBPF` if `KernelModule` is selected, but it doesn't print a warning. Collector also accepts `KernelModule` as a collection method, but switches it to `eBPF`. Here is the commit from the previous PR that added the code to switch `KernelModule` to `eBPF`. https://github.com/stackrox/stackrox/pull/5874/commits/bf283ed122e535bacbd99b9499ab8b2597a8c18c